src/pages/workspaces/workspace/Notebooks.js

```diff
@@ -307,15 +308,14 @@ const Notebooks = _.flow(
       }))
     )(notebooks)
 
-    const getGalaxyText = () => {
-      return app ?
+    const getGalaxyText = (app, galaxyDataDisks) => {
+      const dataDisk = currentDataDisk(app, galaxyDataDisks)
+      return (app && dataDisk) ?
```

Comment:
Added `&& dataDisk` to prevent the error that occasionally happened (probably due to race condition at the first render of the component)